report/main.qmd

```diff
@@ -1 +1,166 @@
+  xlab("Day") + 
+  ylab("Daily percentage of \nzero-calorie beverages sales (%)") +
+  ggtitle("",subtitle = "Figure 2: Daily percentage of zero-calorie beverages sales over time for \neach of the three sites")
+```
+
+
+```{r boxplot}
+#Create a new Intervention column to make boxplots
+sale_data_c$Intervention_new <- sale_data_c$Intervention
+# replace the "preint" in the 2nd half of the study with "preint2"
+# replace the "follow" in the 2nd half of the study with "follow2"
+levels(sale_data_c$Intervention_new) <- c(levels(sale_data_c$Intervention_new), "preint2", "follow2") #first increase the levels of Intervention_new
+
+sale_data_c <- sale_data_c %>%
+  mutate(Intervention_new = replace(Intervention_new, Count>=110 & Intervention_new == "preint", "preint2"), 
+         Intervention_new = replace(Intervention_new, Count>=110 & Intervention_new == "follow", "follow2"))
+
+# reorder the levels
+sale_data_c$Intervention_new <- fct_relevel(sale_data_c$Intervention_new,
+                                            c("preint", "dismes", "dis", "follow", "preint2", "cal", "excer", "both", "follow2", "wash", "wash2"))
+
+#boxplots
+ggplot(data=sale_data_c %>% 
+         filter(Intervention_new!="wash" & Intervention_new !="wash2"), 
+       aes(x=Intervention_new, y=ZeroCal_percent)) +
+  geom_boxplot() + 
+  facet_wrap( ~ Site, ncol = 3, scales = "free_y") + 
+  xlab("Intervention") + 
+  ylab("Daily percentage of \nzero-calorie beverages sales (%)") +
+  ggtitle("",subtitle = "Figure 3: Boxplots of daily percentage of zero-calorie beverages sales \nfor each of the three sites") + 
+  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1))
+
+```
+
+
+## Formal Analysis
+To see which interventions lead to increases in zero-calorie beverages consumption and/or decreases in sugary beverages consumption, it is recommended to use generalized least squares regression (GLS). GLS is similar to linear regression. The key difference is that GLS allows the residuals to be correlated while linear regression assumes that the residuals are independent. For time series data, as the one that we have, observations close to one another may be more likely to be similar than observations that are relatively remote. Therefore, it may not be reasonable to assume that the residuals of a regression model are independent, and so GLS is preferred over linear regression.
+
+It is recommended to fit GLS models separately for each of the three sites, with zero-calorie beverages percentage or sugary beverages percentage as the response variable. In addition to the reason mentioned in the second paragraph of section 3, another advantage of using percentage terms over raw sales is that we just need to fit models for either zero-calorie beverages percentage or sugary beverages percentage instead of for both. An increase in zero-calorie beverages percentage implies a decrease in sugary beverages percentage, vice versa.
```

Comment:
It seems to me that fitting one model per site won't allow you to estimate the marginal average treatment effect (ie, regardless of site); it will also make your conditional effects estimates (given site) less precise. If you want one model per site instead of a single model with potential interaction terms, I think you need to add a bit of justification